Formula/ffmpeg.rb

```diff
@@ -64,6 +68,7 @@ def install
       --enable-libopencore-amrwb
       --enable-librtmp
       --enable-libspeex
+      --enable-nonfree
```

Comment:
This cannot be added as it makes it so we cannot legally distribute `ffmpeg`.

Reply:
Make sense - thanks for the feedback. I'll make the change, but will need to remove fdk since I think it was requiring the non-free flag.

Aside from dependencies, is there any way to "extend" the flags being passed? Options were a great way, but I understand the undertaking to maintain that for you and the other maintainers.

Reply:
Indeed, Fraunhofer has a quite strange policy for its… «open source»!